```
error: linking with `./fake-ld.sh` failed: exit code: 1
[linker command line elided for brevity]
note: /home/servo/.cargo/registry/src/github.com-0a35038f75765ae4/openssl-0.6.4/src/bio/mod.rs:63: error: undefined reference to 'BIO_set_mem_eof_return_shim'
/home/servo/.cargo/registry/src/github.com-0a35038f75765ae4/openssl-0.6.4/src/bio/mod.rs:75: error: undefined reference to 'BIO_eof_shim'
/home/servo/.cargo/registry/src/github.com-0a35038f75765ae4/openssl-0.6.4/src/ssl/mod.rs:452: error: undefined reference to 'SSL_CTX_set_read_ahead_shim'
/home/servo/.cargo/registry/src/github.com-0a35038f75765ae4/openssl-0.6.4/src/ssl/mod.rs:700: error: undefined reference to 'SSL_set_tlsext_host_name_shim'
collect2: error: ld returned 1 exit status

error: aborting due to previous error
Could not compile `b2s`.
```

--- a/ports/gonk/.cargo/config
+++ b/ports/gonk/.cargo/config
@@ -1,6 +1,3 @@
 [target.arm-linux-androideabi]
 ar = "arm-linux-androideabi-ar"
 linker = "./fake-ld.sh"
-
-[target.arm-linux-androideabi.openssl]
-rustc-flags = "-l crypto -l ssl"

--- a/python/servo/command_base.py
+++ b/python/servo/command_base.py
@@ -285,7 +285,7 @@ class CommandBase(object):
                 "%(gonkdir)s/out/target/product/%(gonkproduct)s/obj/lib"
             ) % {"gonkdir": env["GONKDIR"], "gonkproduct": env["GONK_PRODUCT"]}
             env["OPENSSL_LIB_DIR"] = openssl_dir
-            env['OPENSSL_INCLUDE_DIR'] = path.join(openssl_dir, "include")
+            env['OPENSSL_INCLUDE_DIR'] = path.join(env["GONKDIR"], "external/openssl/include")
 
         # FIXME: These are set because they are the variable names that
         # android-rs-glue expects. However, other submodules have makefiles that

